CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [v1.20.0] - 2019-0-30
```

Comment:
Should the version and prospective release date be added to the change log on this PR or only when we cut a release? Not sure myself how godo handles it.

/cc @hilary

Reply:
Going to merge this and cut a release today via instructions in https://github.com/digitalocean/godo/blob/master/CONTRIBUTING.md#releasing

Reply:
Actually:
> Make a pull request with these changes. This PR should be separate from the PR containing the godo changes.
So I'll pull out this changelog update, merge, and then PR separately for the release.